src/App.jsx

```diff
@@ -1,7 +1,23 @@
 import React from 'react';
-import { Trainee } from './pages/Trainee';
+import { BrowserRouter as Router, Route, Redirect,Switch } from 'react-router-dom';
+import { TextFieldDemo, InputDemo, Trainee, ChildrenDemo, Login, NotFound,} from './pages/index';
+import { AuthRoute, PrivateRoute } from './routes/index';
 
 const App = () => (
-  <Trainee />
+  <div>
+    <Router>
+      <Switch>
+        <Route exact path="/">
+          <Redirect to="/Trainee" />
+        </Route>
+        <AuthRoute path="/login" component={Login} />
+        <PrivateRoute path="/ChildrenDemo" component={ChildrenDemo} />
+        <PrivateRoute path="/TextFieldDemo" component={TextFieldDemo} />
+        <PrivateRoute path="/InputDemo" component={InputDemo} />
+        <PrivateRoute path="/Trainee" component={Trainee} />
+        <PrivateRoute component={NotFound} />
```

Comment:
1. Use routes with the axact path
2. routes path will be in small cases
ex - <PrivateRoute exact path="/trainee" component={Trainee} />